alpine-apache/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM smebberson/alpine-base:1.1.0
+MAINTAINER Scott Mebberson <scott@scottmebberson.com>
+
+# Install apache
+RUN apk add --update apache2 apache2-utils && \
+    rm -rf /var/cache/apk/* && \
+    ln -sf /dev/stdout /var/log/apache2/access.log && \
```

Comment:
I plan on upgrading `alpine-consul-nginx` to not automatically stream logs to `stdout` and `stderr` as per `alpine-nginx`. It's an easy two lines in a downstream `Dockerfile` to stream them if you wish. What do you think @matthewvalimaki, should the `alpine-apache` image work the same?